valkey.conf

```diff
@@ -386,6 +386,11 @@ databases 16
 # ASCII art logo in startup logs by setting the following option to yes.
 always-show-logo no
 
+# To avoid logging personal identifiable information (PII) into server log file,
+# uncomment the following:
+#
+# hide-client-log yes
```

Comment:
```suggestion
# hide-user-data-in-log yes
```
Trying to come up with a more descriptive name.

Reply:
i see Redis is using `hide-user-data-from-log`, do we need to adopt this name?

Reply:
Lol, maybe we shouldn't take their name so it's more obvious it's not because we looked at their code.